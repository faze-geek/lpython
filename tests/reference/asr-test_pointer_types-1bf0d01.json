{
    "basename": "asr-test_pointer_types-1bf0d01",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/test_pointer_types.py",
    "infile_hash": "750a2bb08545b8c11e44585855545060693c59a71df5989ac5371720",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-test_pointer_types-1bf0d01.stderr",
    "stderr_hash": "6c69f3c82186fa0f2cfe47b5a3d03de7448ed7db33ab0d27f6254858",
    "returncode": 2
}