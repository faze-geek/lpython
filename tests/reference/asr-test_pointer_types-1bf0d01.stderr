semantic error: 'i32[3]' object does not support item assignment
 --> tests/errors/test_pointer_types.py:6:5
  |
6 |     y[0] = 1
  |     ^^^^^^^^ 
